raiden/waiting.py

```diff
@@ -281,7 +281,7 @@ def wait_for_close(
     )
 
 
-def wait_for_payment_network(
+def wait_for_token_network(
```

Comment:
nitpick: perhaps add a small docstring explaining what part of a token network is waited on.

Reply:
Does this look sensible? https://github.com/raiden-network/raiden/pull/4517/commits/287c65655de4bb0009949e0137a637e96b36f661